src/index.ts

```diff
@@ -88,10 +89,7 @@ export {
     CommandInvocation,
 } from "./internal/invoker/Payload";
 
-export {
-    logger,
-    LoggingConfig,
-} from "./internal/util/logger";
+export * from "./internal/util/logger";
```

Comment:
Seems odd to export stuff from under internal. Perhaps this does not belong under internal?

Reply:
That's right. I'll move it to `util`.